.github/workflows/continuous-integration.yml

```diff
@@ -13,6 +13,8 @@ jobs:
         key: ${{ runner.os }}-node_modules-${{ hashFiles('yarn.lock') }}
     - name: Install
       run: yarn install --frozen-lockfile
+    - name: Set up frontend-shared
+      run: yarn setup-frontend-shared
```

Comment:
transpiles the src files and sets up yarn link.
doing this step each time has only a small performance cost and the stdout, which normally tells you its already linked, its just hidden.

 I don't think we actually need to add @hypothesis/frontend-shared in our package.json because of the yarn link, but we will need to do that as soon as we move to its own repo.